modules/templates/helper.go

```diff
@@ -131,7 +132,10 @@ func NewFuncMap() template.FuncMap {
 		"DisableImportLocal": func() bool {
 			return !setting.ImportLocalPaths
 		},
-		"DefaultTheme": func() string {
+		"ThemeName": func(userTheme any) string {
+			if reflect.ValueOf(userTheme).Kind() == reflect.String && userTheme != "" {
+				return userTheme.(string)
+			}
 			return setting.UI.DefaultTheme
```

Comment:
Maybe there are better ways than `reflect`. The function receives either nil or a string.